src/pcre2_compile_class.c

```diff
@@ -2125,6 +2155,7 @@ switch (meta)
     pop_info->length = 1;
     *code_start = pop_info->op_single_type = ECL_ANY;
     memset(pop_info->bits.classbits, 0xff, 32);
+    context->needs_bitmap = TRUE;
```

Comment:
I believe that this OP_ALLANY → ECL_ANY branch does not need to set needs_bitmap.

Reply:
True!